tests/copy.earth

```diff
@@ -222,3 +222,9 @@ copy-empty-src:
 copy-empty-src-if-exists:
     ARG SRC=""
     COPY --if-exists $SRC $SRC
+
+copy-if-exists-wildcard:
+    COPY --if-exists non_existing_folder/* /
```

Comment:
what about adding:

```
ARG test_foo_exists=false
IF [ "$test_foo_exists" ]
    RUN test -f touch non_existent_folder/foo
END
```

so we can then call:

```
DO +RUN_EARTHLY --earthfile=copy.earth --target="+copy-if-exists-wildcard --test_foo_exists=true"
```

and perhaps another similar arg for testing the expected outcome when `non_existent_folder` exists but is empty?

Reply:
Updated with `test -f`. I used separate targets as the parameters were more complicated than the extra 3 lines.